src/reports/resources/demo.js

```diff
@@ -653,6 +653,10 @@ function get_progress(loc) {
         if (req2.readyState == 4) {
             if (req2.status == 202) {
                 document.getElementById("progress").textContent = req2.responseText;
+                const nums = document.getElementById("num-jobs");
+                if (nums != null) {
+                    nums.textContent = req2.getResponseHeader("X-Job-Count");
+                }
```

Comment:
This is fine but why the conditional? Does this element ever go away?

Reply:
Ya if --demo isn't passed in you get an object not found error in the console.